ci/views.py

```diff
@@ -586,15 +590,36 @@ def identity(request, identity):
                         extra_tags='success'
                     )
 
+            elif 'optout_identity' in request.POST:
+                details = results.get('details', {})
+                addresses = details.get('addresses', {})
+
+                for address_type, addresses in addresses.items():
+                    for address, info in addresses.items():
+                        idApi.create_optout({
```

Comment:
Does the hellomama registration app have an endpoint that enables opt outs through there? If so then it's probably better to do the optout through there. That would allow project-specific business logic to be executed on optout (like momconnect sending the optout data to jembi).